Topic3TDD/src/exc2/WordWrapperTest.java

```diff
@@ -0,0 +1,44 @@
+package exc2;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+public class WordWrapperTest {
+
+	@Test
+	public void firstExampleTest() {
+		List<String> expected = new ArrayList<String>();
+		expected.add("Hello!");
+		assertEquals(expected, WordWrapper.wrapper("Hello!", 60));
+	}
+
+	@Test
+	public void secondExampleTest() {
+		List<String> expected = new ArrayList<String>();
+		expected.add("Hello");
+		expected.add("World!");
+		assertEquals(expected, WordWrapper.wrapper("Hello World!", 6));
+	}
+
+	@Test
+	public void thirdExampleTest() {
+		List<String> expected = new ArrayList<String>();
+		expected.add("a b");
+		expected.add("c d");
+		expected.add("e f");
+		assertEquals(expected, WordWrapper.wrapper("a b c d e f", 3));
+	}
+
+	@Test
+	public void fourExampleTest() {
+		List<String> expected = new ArrayList<String>();
+		expected.add("Excel");
+		expected.add("ent");
+		assertEquals(expected, WordWrapper.wrapper("Excelent", 5));
+	}
+
+}

```

Comment:
Nice work, I liked the way your handled the Wrapper method, clean and concise and you used "StringTokenizer" to help you to work with the words :+1